scripts/.env

```diff
@@ -0,0 +1,14 @@
+# Env variables for docker-prover-monitor
+
+# For error that is matched, return the previous and after N number of lines.
+N_LINES_BEFORE=3
+N_LINES_AFTER=3
+
+# Every X number of seconds to check the logs.
+CHECK_INTERVAL_SECS=60
+
+# The slack channel alert messages are sent.
+CHANNEL="#docker-prover-monitor-alerts"
+
+# Token for slack bot for alert messages. Replace with docker-prover-monitor bot token.
+BOT_AUTH_TOKEN="xoxb-REPLACE-WITH-SLACK-BOT-TOKEN-FOR-DOCKER-PROVER-MONITOR"
```

Comment:
For this method, as it is public repo and will share to anyone, so maybe such auth token will be not a good method to share.
Maybe need check whether have any other method like public key to share to allow them send to the slack channel.